pkg/controller/keycloakrealm/keycloakrealm_reconciler_test.go

```diff
@@ -219,7 +219,7 @@ func TestKeycloakRealmReconciler_Update(t *testing.T) {
 	// 1 - update realm
 	assert.IsType(t, &common.PingAction{}, desiredState[0])
 	assert.IsType(t, &common.UpdateRealmAction{}, desiredState[1])
-	assert.Len(t, desiredState, 2)
+	assert.Len(t, desiredState, 3)
```

Comment:
Can you improve this test coverage? Along with the new length, you can inspect `desiredState[2]` like is done on line `220` and `221`. Can you potentially also verify an actual change is properly handled? It looks like that is what the rest of the checks after this line do, but I haven't looked into this portion too closely for my own work yet.